modulus/models/afno/afno.py

```diff
@@ -340,26 +340,29 @@ class PatchEmbed(nn.Module):
 
     Parameters
     ----------
-    img_size : Tuple[int, int]
-        Input image dimensions (height, width)
+    inp_shape : List[int]
+        Input image dimensions [height, width]
     in_channels : int
         Number of input channels
-    patch_size : Tuple[int, int], optional
-        Size of image patches, by default (16, 16)
+    patch_size : List[int], optional
```

Comment:
I feel like you mentioned this to me but why the update to List here?

Reply:
This is because hydra/yaml can't take Tuple